blosc/blosc2.c

```diff
@@ -820,8 +821,6 @@ uint8_t* pipeline_forward(struct thread_context* thread_context, const int32_t b
 
   /* Prefilter function */
   if (context->prefilter != NULL) {
-    /* Set unwritten values to zero */
-    memset(_dest, 0, bsize);
```

Comment:
Is this ok? @FrancescAlted

Reply:
I confirm that this line is necessary, so you need to add it again.

Reply:
solved at f2fdffa71814dd1cbda6d3388e551c7e61214f64